packages/ckeditor5-clipboard/src/dragdropexperimental.ts

```diff
@@ -632,18 +644,27 @@ export default class DragDropExperimental extends Plugin {
 			} );
 
 			global.document.body.appendChild( this._previewContainer );
-		} else {
-			this._previewContainer.removeChild( this._previewContainer.firstElementChild! );
+		} else if ( this._previewContainer.firstElementChild ) {
+			this._previewContainer.removeChild( this._previewContainer.firstElementChild );
+		}
+
+		const domRect = new Rect( domEditable );
+
+		// If domTarget is inside editable root, browser display preview by own.
```

Comment:
```suggestion
		// If domTarget is inside the editable root, browsers will display the preview correctly by themselves.
```

I think this reads better.